### PR TITLE
 convert bruto to neto salary

### PR DESCRIPTION
**Pull request by berda97**

### CODE REVIEW

File: AccountingService/Services/ExchangeRateService.cs [hunk 1]
```diff
@@ -30,7 +30,7 @@ private static void ReloadExchangeRates()
 
             var request = new RestRequest();
             request.Method = Method.Get;
-            request.AddHeader("apikey", "[API-KEY]");
+            request.AddHeader("apikey", "6IRdNpTdHNvBw5bd4ouVVoAgo5WpZA2E");
```
Comment (ghost) [inline AccountingService/Services/ExchangeRateService.cs]: Please remove this API key. We don't want to leak secrets!

File: AccountingService/Services/NetSalaryService.cs [hunk 2]
```diff
@@ -0,0 +1,23 @@
+﻿namespace AccountingService.Services
+{
+    public class NetSalaryService
+    {
+        const double taxRate = 0.2;
+        const double socialSecurityRate = 0.1;
+        const double pensionRate = 0.05;
+        public double Calculate( double grossSalary)
+        {
+
```
Comment (ghost) [inline AccountingService/Services/NetSalaryService.cs]: Remove extra spacing

File: AccountingService/Services/NetSalaryService.cs [hunk 3]
```diff
@@ -0,0 +1,23 @@
+﻿namespace AccountingService.Services
+{
+    public class NetSalaryService
+    {
+        const double taxRate = 0.2;
```
Comment (Berdic) [inline AccountingService/Services/NetSalaryService.cs:5]: Move this to configuration file. We want to easily adjust rates if needed.

**Review by ghost — COMMENTED** [review]
Looks good. There are few things to improve.

File: AccountingService/Services/ExchangeRateService.cs (hunk 1 above)
Comment (berda97) [reply AccountingService/Services/ExchangeRateService.cs]: done!

File: AccountingService/Services/NetSalaryService.cs (hunk 2 above)
Comment (berda97) [reply AccountingService/Services/NetSalaryService.cs]: done!

**Review by ghost — APPROVED** [review]
LGTM!

**Pull request merged**